src/solve/utils.ts

```diff
@@ -0,0 +1,29 @@
+import { Asset, AssetWithMetadata } from "./types";
+
+export function assetHasMetadata(asset: Asset) {
```

Comment:
Currently the API returns a lot of assets that are not necessarily "assets", like osmosis pool tokens (see: https://api.skip.money/v1/fungible/assets?chain_id=osmosis-1&native_only=true). 

Do we want this? (will post in slack)

Either way for now this seems to be the best way to filter them out of the UI.